Styling/styles/styles.css

```
@@ -1312,7 +1316,8 @@ body.articles, body.tutorials {
         @apply my-4;
       }
       pre code {
-        @apply whitespace-pre-wrap text-xs font-extralight p-4 bg-gray-900 text-white block my-4 lg:text-base lg:p-8;      
+        @apply whitespace-pre-wrap text-xs font-extralight p-4 bg-gray-900 text-white block my-4 lg:text-base lg:p-8;
+        overflow-x: auto;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)